server/portal/apps/system_monitor/unit_test.py

```diff
@@ -1,72 +1,44 @@
 import pytest
 import json
 import os
-import pytz
-from datetime import datetime
 from django.conf import settings
 from django.http import Http404
 
 
 @pytest.fixture
-def system_status_old(scope="module"):
+def system_status(scope="module"):
     yield json.load(open(os.path.join(settings.BASE_DIR, 'fixtures/system_monitor/index.json')))
 
 
-@pytest.fixture
-def system_status(system_status_old, scope="module"):
-    # alter time stamps so that the system status looks like it was collected recently
-    altered_system_status = system_status_old.copy()
-    system = altered_system_status['frontera.tacc.utexas.edu']
-    for test_entry in system['tests']:
-        system['tests'][test_entry]['timestamp'] = str(pytz.utc.localize(datetime.now()))
-    yield altered_system_status
-
-
 @pytest.fixture
 def system_status_missing_frontera(scope="module"):
     yield json.load(open(os.path.join(settings.BASE_DIR, 'fixtures/system_monitor/index_missing_frontera.json')))
 
 
 @pytest.mark.django_db()
 def test_system_monitor_get(client, settings, requests_mock, system_status):
-    settings.SYSTEM_MONITOR_DISPLAY_LIST = ['frontera.tacc.utexas.edu']
+    settings.SYSTEM_MONITOR_DISPLAY_LIST = ['Frontera']
     requests_mock.get(settings.SYSTEM_MONITOR_URL, json=system_status)
     response = client.get('/api/system-monitor/')
     assert response.status_code == 200
     system = response.json()[0]
-    assert system['hostname'] == 'frontera.tacc.utexas.edu'
```

Comment:
Should we keep this assertion?

Reply:
Updated to include this assertion as well.